### DISCUSSION TITLE
Option for multiple segment labels

### DISCUSSION THREAD

**Discussion by juliussss** [mifi/lossless-cut#1145] [Ideas]
The purpose of multiple labels here is for the export + merge cuts workflow.

Imagine a scenario where you want to add multiple labels to the same segment cut and be able to export and merge the segments for each given label e.g. 

Segment 91
01:30:00.000-01:35:00.000
Labels: ["Scene 2", "Review", "Angle 5"]

EXPORT + MERGE CUTS

some-video-scene_2_merged.mp4
some-video-review_merged.mp4
some-video-angle_5_merged.mp4

Each merged export file should include Segment 91.

**Comment by mifi** [comment]
I think what you are asking is related to #812

**Reply by juliussss** [reply]
Thanks for acknowledging and you are right, I hadn't seen that issue.

In my use case the "groups" could be defined via multiple label support but export + merge would be identical.
